scipy/optimize/tests/test_milp.py

```diff
@@ -273,6 +273,46 @@ def test_infeasible_prob_16609():
     np.testing.assert_equal(res.status, 2)
 
 
+_msg_time = "Time limit reached. (HiGHS Status 13:"
+_msg_iter = "Iteration limit reached. (HiGHS Status 14:"
+
+
+@pytest.mark.skipif(np.intp(0).itemsize < 8, reason="Unhandled 32-bit GCC FP bug")
```

Comment:
```suggestion
@pytest.mark.skipif(np.intp(0).itemsize < 8,
                    reason="Unhandled 32-bit GCC FP bug")
```

Reply:
I'll dig deeper into the `prerelease_deps_coverage_64bit_blas` failure this evening.  A cursory reading of test logs indicate the time out test did indeed report a time out status (though it's not clear if it did after 1 second as desired) but failed to find a solution and the node limited test had pytest time it out (`Failed: Timeout >60.0s`)

Reply:
When I try to follow the azure-travis-template.yml steps for this job locally in a VM, I fail to reproduce the errors.  I get another unrelated test failure (`scipy/sparse/linalg/_dsolve/tests/test_linsolve.py::TestFactorized::test_bug_8278`), but these milp tests run fine.

Any ideas on what to try next to reproduce/resolve this?